mackerel/api_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"net/http/httptest"
 	"reflect"
 	"testing"
+
+	mkr "github.com/mackerelio/mackerel-client-go"
```

Comment:
[imo]
I don't prefer `mkr` as the alias of mackerel-client-go, since we have a product named `mkr`…

Reply:
I agree but can't help until we have mackerel package here.

Reply:
😭